dataactvalidator/config/sqlrules/c12_cross_file.sql

```diff
@@ -12,4 +12,8 @@ WHERE ap.submission_id = {}
 		WHERE af.submission_id = ap.submission_id
 		    AND af.piid = ap.piid
 		    AND af.parent_award_id IS NOT DISTINCT FROM ap.parent_award_id
+	) AND NOT EXISTS(
+	  SELECT transaction_obligated_amou
+	  FROM award_financial AS af
+	  WHERE transaction_obligated_amou IS NULL
```

Comment:
I'm not the best at SQL so maybe this wouldn't work, but why not just add `AND af.transaction_obligated_amou IS NULL` to the NOT EXISTS above?